src/paddle.ts

```diff
@@ -12,7 +12,8 @@ export const isPlusMember = (cycle: SubscriptionCycles | undefined): boolean =>
   !!cycle?.length || false;
 
 export const isGiftedPlus = (
-  subscriptionFlags: UserSubscriptionFlags,
```

Comment:
let's keep it required if that is ok, you can early return in the query if `subscriptionFlags` is undefined?

Reply:
Done ✔️

Reply:
Why are gifterId and cycle still required, they can technically be undefined and then `isGiftedPlus` would return false no?

Reply:
They are not required in params, but if the guard function pass the return an `UserSubscriptionFlags` object that have mandatory `gifterId` and `cycle`. 

This is helping to simplify the code where this function is used since now ts knows that we have already checked for those properties!

Reply:
ahh nice in fact, I missed the `is` part 🚀